packages/browser/examples/assets/config/widget/server/sparql_server.json

```diff
@@ -1,4 +1,76 @@
 {
-  "url": "http://localhost:8997/strabon/",
```

Comment:
config mixes widget aspect config + strabon server config
split config in two config files:
one called "strabon.json" in assets/config/widget/server/
one called "sparql.json" in assets/config/widget
could be nice to move folder assets/config/widget/server in assets/config/server since a server config could be not related to a widget

Reply:
the config has been separated into 
- `assets/config/widget/server/sparql_server.json`
- `assets/config/widget/server/sparql_widget.json`